modules/gatk4/variantfiltration/main.nf

```diff
@@ -19,14 +19,14 @@ process GATK4_VARIANTFILTRATION {
     }
 
     input:
-    tuple val(meta), path(vcf)
+    tuple val(meta), path(vcf), path(vcf_index)
```

Comment:
```suggestion
    tuple val(meta), path(vcf), path(vcf_tbi)
```

We discussed trying to standaradise naming on monday and settled on tbi https://github.com/nf-core/modules/pull/1074/files#diff-c0d34d7885b819641586baffb1a7a05dc3832a955f461d3937a2b984522d2c5e

Reply:
Sounds good. I can make changes and submit again.

Reply:
Done!